dist/images/Dockerfile

```diff
@@ -8,25 +8,29 @@
 # This is for a development build where the ovn-kubernetes utilities
 # are built in this Dockerfile and included in the image (instead of the rpm)
 #
-# This is based on centos:7
+# This is based on centos:8
```

Comment:
why are we still supporting centos? shouldn't we just remove this Dockerfile and move the fedora one to be the main one?

Reply:
+1  I asked the same question here -> https://coreos.slack.com/archives/GQ0CU2623/p1644507883910489 

The verdict was to keep it, but we can remove it if you want

Reply:
centos is EOL, lets jsut remove it

Reply:
Sounds good I'll do that now and rebase as well